src/shared/sources/pl/index.js

```diff
@@ -0,0 +1,84 @@
+const parse = require('../_lib/parse.js')
+const maintainers = require('../_lib/maintainers.js')
+
+const isoMap = require('./mapping.json')
+
+const country = "iso1:PL"
+
+module.exports = {
+    country,
+    maintainers: [ maintainers.qgolsteyn, maintainers.ciscorucinski ],
+    friendly: {
+        name: 'Ministry of Health of the Republic of Poland',
+        url: 'http://www.mz.gov.pl/'
+    },
+    scrapers: [
+        // TODO (scraper) PL: 2020-03-18 scraper
+        // {
+        //     startDate: '2020-03-18',
+        //     crawl: [
+        //         {
+        //             type: 'csv',
+        //             url: 'https://raw.githubusercontent.com/covid19-eu-zh/covid19-eu-data/master/dataset/covid-19-pl.csv'
+        //         }
+        //     ],
+        //     scrape (data, date) {
+        //         const casesByRegion = {}
+        //         const deathsByRegion = {}
+        //
+        //         for (const item of data) {
+        //             const {datetime, nuts_2} = item;
+        //             if (datetime.dateIsBeforeOrEqualTo(datetime, date) && nuts_2) {
+        //                 casesByRegion[nuts_2] = parse.number(item.cases)
+        //                 deathsByRegion[nuts_2] = parse.number(item.deaths)
+        //             }
+        //         }
+        //
+        //         const states = []
+        //         for (const region of Object.keys(casesByRegion)) {
+        //             states.push({
+        //                 state: mapping[region],
+        //                 cases: casesByRegion[region]
+        //             })
+        //         }
+        //
+        //         if (states.length > 0) states.push(transform.sumData(states))
```

Comment:
transform doesn't exit (that I am aware of) in Li. How should this be refactored?

Reply:
It's in `const transform = require(srcShared + 'sources/_lib/transform.js')` -- was it not working for you?

Reply:
Added, works ok!